slave/slave_channels.go

```diff
@@ -0,0 +1,43 @@
+package slave
+
+import (
+	"fmt"
+
+	"github.com/aaronang/cong-the-ripper/lib"
+)
+
+func addTask(task lib.Task, s *Slave) {
```

Comment:
You can also use the Slave as the receiver even if it's in a different file. Generally Go source files are pretty big, and they're structured like this: types at the top, Public methods afterwards and private methods at the bottom. But you're welcome to keep it this way, it's only a cosmetic issue.